src/index.html

```diff
@@ -5,9 +5,267 @@
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__top">
+        <div class="header__logo logo"></div>
+        <div class="header__phone">
+          <a href="tel:+1 234 555-55-55" class="header__link">+1 234 555-55-55</a>
+        </div>
+        <div class="header__action">
+          <a href="#" class="header__link">call to order</a>
+        </div>
+      </div>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item--is-active">
+            Home
+          </li>
+          <li class="nav__item">
+            <a href="#about" class="nav__link">About Us</a>
+          </li>
+          <li class="nav__item">
+            <a href="#gallery" class="nav__link">Gallery</a>
+          </li>
+          <li class="nav__item">
+            <a href="#consultation" class="nav__link">Consultation</a>
+          </li>
+          <li class="nav__item">
+            <a href="#testimonials" class="nav__link">Testimonials</a>
+          </li>
+          <li class="nav__item">
+            <a href="#contacts" class="nav__link">Contacts</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="section section--white" id="about">
+        <div class="info section__info">
+          <h1 class="info__title">
+            Miami
+            <strong>best</strong>
+            <br>
+            Real estate
+            <a href="#" class="info__link">here</a>
+          </h1>
+          <div class="info__text">
+            <p>
+              <strong>Over 50 years experience in love.</strong>
+              <br>
+               Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism,
+              <br>
+               which originated in the music of the early twentieth century, microforms,
+              <br>
+               found a distant historical.
+            </p>
+
+            <p>
+              The market leader in real estate since 2003.
+              <br>
+              Awards Real Estate Company, 2010, 2011, 2013.
+              <br>
+              More than 200 satisfied customers premium segment.
+              <br>
+              We focus only on new buildings and high-level doskanalno know this area better than other specialists.
+              <br>
+              Working directly with developers - so have more useful information about the profitable deals.
+            </p>
+
+            <p>
+              We will never be late!
+              <br>
+              Guaranteed secure transaction - 100%.
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="section" id="gallery">
+        <h2 class="heading section__heading">Gallery <span class="heading__crown"></span></h2>
+        <div class="gallery">
+          <div class="gallery__photos photos">
+            <img
+              src="images/photos/1.jpg"
+              alt="Gallery-Photo"
```

Comment:
Прибери дефіси, це буде читати скрінрідер
```suggestion
              alt="Gallery-Photo"
```